doc/whats_new/v0.20.rst

```diff
@@ -303,3 +303,8 @@ Changes to estimator checks
 - Allow :func:`estimator_checks.check_estimator` to check that there is no
   private settings apart from parameters during estimator initialization.
   :issue:`9378` by :user:`Herilalaina Rakotoarison <herilalaina>`
+
+- Added :func:`estimator_checks.check_set_params` test which checks that
```

Comment:
This link won't work as we don't generate docs for estimator checks (though perhaps we should)